app/services/vot/component_value.rb

```diff
@@ -1,5 +1,34 @@
 # frozen_string_literal: true
 
 module Vot
-  ComponentValue = Data.define(:name, :description, :implied_component_values, :requirements).freeze
+  ComponentValue = Data.define(
```

Comment:
i think i'd rather we just revert this to be the class-like object for the purposes of this PR, just to keep it a little more straightforward to review. is that ok? if you feel really strongly about it, we can keep it as-is though.